CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```

Comment:
I'm curious about the Changelog changes. Typically the changelog only includes user-facing changes and it is generated with the `update-changelog` script during the RC phase :thinking: for this reason I think we should not include these changes here?

Reply:
I removed some entries that weren't user-facing. Thanks for pointing that out! 

All of the remaining entries are major changes for package-level exports. See our documentation on [reviewing release PRs](https://github.com/MetaMask/core/blob/main/docs/reviewing-release-prs.md?plain=1#L100-L110) for how these qualify as breaking changes. 

As for why these are in the changelog file, the shared libraries team has been recording changelog entries for new PRs under the "Unreleased" heading, so that they're immediately available for reference when the next release is being created.

`update-changelog` auto-populates a list of unreleased commits to changelog, but [these are only intended as starting points](https://github.com/MetaMask/core/blob/main/docs/reviewing-release-prs.md?plain=1#L159). Recording under "Unreleased" streamlines the process of retrieving the full changelog entries by making it unnecessary to visit each linked PR and check the changelog section of the PR description.